src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/Example1.java

```diff
@@ -24,15 +24,16 @@ void test() {
   abstract interface I {
     public abstract void m();
     // 2 violations above:
-    //                   'Redundant 'public' modifier'
-    //                   'Redundant 'abstract' modifier'
+    //    'Redundant 'public' modifier'
+    //    'Redundant 'abstract' modifier'
```

Comment:
because of https://github.com/checkstyle/checkstyle/pull/15379#discussion_r1693661024